pkg/build/builder/util.go

```diff
@@ -192,3 +194,10 @@ func reportPushFailure(err error, authPresent bool, pushAuthConfig docker.AuthCo
 	}
 	return fmt.Errorf("Failed to push image: %v", err)
 }
+
+// addBuildLabels adds some common image labels describing the build that produced
```

Comment:
actually it's correct as is.  what's incorrect is "this time" on the next line which should read "this image"